src/config.ts

```diff
@@ -322,6 +322,28 @@ export const BUNDLE_REPAIR_RETRY_BATCH_SIZE = +env.varOrDefault(
   '1000',
 );
 
+//
+// PostgreSQL
+//
+
```

Comment:
This is probably not the exact right location to add them, but let's add `CHUNK_DATA_SOURCE_TYPE` and `CHUNK_METADATA_SOURCE_TYPE` environment variables to configure which chunk sources to use.

Reply:
done